package.json

```diff
@@ -56,27 +51,13 @@
     "js-yaml": "3.13.1",
     "json-to-graphql-query": "^2.2.0",
     "jsonata": "^1.8.5",
+    "jsonwebtoken": "^8.5.1",
```

Comment:
Are we not able to use `jose` instead of importing another jwt package?

Reply:
unfortunately not: we are loading the Grant config synchronously, and jose is only able to sign JWTs asynchronously. However we could consider replacing `jose` by `jsonwebtoken` - but it needs further thinking and it's better to leave it as it is for now